pages/my-collections.tsx

```diff
@@ -0,0 +1,122 @@
+import Image from 'next/image';
+import withTransition from '../components/withTransition';
+import { ConnectToWalletBanner } from '../components/ConnectToWalletBanner';
+import { Loader, Banner } from '../components';
+import { addressToCreatorImage, shortenAddress } from '../utils/address';
+import { useAlephiumConnectContext } from '@alephium/web3-react';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import Link from 'next/link';
+import { motion } from 'framer-motion';
+import { shortenName } from '../utils';
+
+interface NFTCollection {
+  _id: string  // NFTCollection contract id
+  type: 'NFTOpenCollection' | 'NFTPublicSaleCollection'
+  owner: string
+  name: string
+  description: string
+  image: string
+}
+
+const NFTCollectionCard = ({ collection }: { collection: NFTCollection }) => {
+  return (
+    <Link href={{ pathname: '/collection-details', query: { collectionId: collection._id } }}>
+      <motion.div
+        whileInView={{ opacity: [0, 1] }}
+        transition={{ duration: 0.5 }}
+        className="dark:bg-nft-black-3 bg-white rounded-2xl cursor-pointer shadow-md hover:shadow-lg duration-500 p-4"
+      >
+        <div className="relative w-full h-52 sm:h-48 minmd:h-60 minlg:h-280 rounded-2xl overflow-hidden">
+          <Image
+            className="flex justify-center items-center hover:scale-110 transition-all duration-500"
+            src={collection.image}
+            layout="fill"
+            objectFit="cover"
+            alt={`nftCollection${collection._id}`}
+          />
+        </div>
+        <div className="mt-3 flex flex-col">
+          <p className="font-poppins dark:text-white text-nft-black-1 font-semibold text-xs minlg:text-xl">
+            {collection.name.length > 14 ? shortenName(collection.name) : collection.name}
```

Comment:
[Nit] Perhaps we can show the `totalSuppy` in the card? Similar to extension wallet.